documentation/public/user-guide/inputs/multiple-data-types.sol

```diff
@@ -0,0 +1,4 @@
+struct Foo { uint a; }
```

Comment:
nit suggestion: newlines between types for readability.

Reply:
This bloats up the space in the example in the docs and I don't find it more readable, personally. Could we skip it, please?

Reply:
Sounds good. Thanks.